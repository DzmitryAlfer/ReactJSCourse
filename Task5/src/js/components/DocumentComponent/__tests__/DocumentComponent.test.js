import React from 'react';
import Renderer from 'react-test-renderer';
import {DocumentComponent} from '../index';

jest.mock('../../header/DocumentHeaderComponent');
jest.mock('../../body/DocumentBodyComponent');

describe('DocumentComponent', () => {
    test('Component render', () => {

        const tree = Renderer.create(
            <DocumentComponent/>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});