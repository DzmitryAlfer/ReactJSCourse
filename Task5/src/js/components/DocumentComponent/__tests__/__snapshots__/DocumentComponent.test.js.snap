// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentComponent Component render 1`] = `
Array [
  <span
    onSearchClick={[Function]}
  >
    DocumentHeaderComponent MOCK
  </span>,
  <span
    onItemClick={[Function]}
    searchString=""
  >
    DocumentBodyComponent MOCK
  </span>,
]
`;
